Resources/Maps/northway.yml

```
@@ -10005,10 +10065,10 @@
           showEnts: False
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)